### PR TITLE
test(help,link,loader,menulist): update cypress with a new approach URL

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Apply new `url` approach in tests for:
`help`  (timing  `1:17` ->  `0:22`),
`link`  (timing  `3:05` -> `0:32` ),
`loader`  (timing  `0:39` -> `0:11` ),
`menuList` (timing  `0:58` ->  `0:36` ).

### Current behaviour

Currently we are passing all parameters via `knobs`.

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/loader.feature [hunk 1]
```diff
@@ -1,46 +1,41 @@
 Feature: Loader default component
   I want to change Loader component properties
```
Comment (DlgSHi) [inline cypress/features/regression/loader.feature:2]:
```suggestion
  I want to test Loader component properties
```

File: cypress/features/regression/menuList.feature [hunk 2]
```diff
@@ -1,72 +1,73 @@
 Feature: MenuList component
   I want to change MenuList component default properties
```
Comment (DlgSHi) [inline cypress/features/regression/menuList.feature:2]:
```suggestion
  I want to test MenuList component default properties
```

File: cypress/locators/index.js [hunk 3]
```diff
@@ -66,7 +66,7 @@ export const commonButtonPreviewNoIframe = () => cy.get(STORY_ROOT).find('button
 export const backgroundUILocatorNoIFrame = () => cy.get(BACKGROUND_UI_LOCATOR);
 export const closeIconButton = () => cy.get(CLOSE_ICON_BUTTON);
 export const fieldHelpPreviewNoIFrame = () => cy.get(FIELD_HELP_PREVIEW).first();
-export const helpIconNoIFrame = () => cy.get(HELP_ICON_PREVIEW).first();
+export const helpIconNoIFrame = () => cy.get(HELP_ICON_PREVIEW);
```
Comment (DlgSHi) [inline cypress/locators/index.js:69]: shouldn't it be `helpIconNoIFrame` -> `helpIcon`  as we discussed?

File: cypress/locators/menu-list/locators.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 // component preview locators
 export const MENU_LIST_CHILDREN = 'div[data-component="menu-list"][class="carbon-menu-list"]';
-export const MENU_TITLE = '#story-root > div[data-component="menu-list"] > [data-component="link"]';
+export const MENU_TITLE = '[data-element="title"]';
```
Comment (DlgSHi) [inline cypress/locators/menu-list/locators.js:3]: I think we already have one locator in `common` locators

Comment (KatarzynaQA) [reply cypress/locators/menu-list/locators.js]: I've refactored it, instead of `'[data-element="title"]'` I've used `getDataElementByValue`

**Pull request merged**